hw10/comparison.py

```diff
@@ -0,0 +1,12 @@
+class User:
+    def __init__(self, user):
+        self.user = user.lower()
```

Comment:
You need to create a `name` attribute here and use it in __eq__:
```
return self.name.lower() == other.name.lower()
```

The difference is that you don't change initial value but transform it when comparing two objects